### ISSUE TITLE
Mentioning uid/gid once in twistd.1 is enough

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5125]
|<img alt="chris-'s avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| chris- reported|
|-|-|
|Trac ID|trac#5125|
|Type|enhancement|
|Created|2011-05-29 11:14:42Z|

Currently, the man page for twistd mentions uid/gid twice, that's unnecessary.

Attachments:

* [twistd.1.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/9ec/9ec169198ed9589be7ec3b9d9096e8a67edf2980/02779ff95ea54fe9218562cfc9c4bb71bedfdc7c.patch) (440 bytes) - added by chris- on 2011-05-29 11:14:56Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5125 5125
type__enhancement enhancement
reporter__chris_ chris-
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1306667682000000 1306667682000000
changetime__1306853385000000 1306853385000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @exarkun|
|-|-|

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

(In [32002]) Apply patch to remove duplicate --uid/--gid man page docs

Author: chris-
Reviewer: exarkun
Fixes: [#5125](https://github.com/twisted/twisted/issues/5125)

Remove the duplicate --uid and --gid documentation from the twistd man page.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

there was also [#4619](https://github.com/twisted/twisted/issues/4619).
